src/mongo/MongoTripodConfig.class.php

```diff
@@ -316,22 +346,70 @@ protected function loadConfig(Array $config)
             $this->tableSpecs[$storeName] = array();
             foreach ($tableSpecs as $spec)
             {
-                if(!isset($spec[_ID_KEY]))
-                {
-                    throw new MongoTripodConfigException("Table spec does not contain " . _ID_KEY);
-                }
+                $this->validateTableSpec($spec);
 
-                // Get all "fields" in the spec
-                $fieldsInTableSpec = $this->findFieldsInTableSpec('fields', $spec);
-                // Loop through fields and validate
-                foreach($fieldsInTableSpec as $field)
+                if(isset($spec['to_data_source']))
```

Comment:
No, because the validation of the table spec stuff is independent of the configuration as a whole.